cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,123 @@
+We also offer support via email with our [paid plans](https://www.cypress.io/pricing/).
+
+- **Feature request for Cypress Dashboard** - Thank you for your support! The issue has been routed to our dashboard team's ticketing system and your customer success representative will be able to follow up with you directly via email.
+
+- **The fix or feature is not within our vision for Cypress** - There will inevitably be suggestions that will not fit within the scope of Cypress' vision for our product. We will do our best to explain why we will not be addressing this issue.
+
+- **It's a dupe** - The issue you have entered has already been logged by another person. We will link the appropriate ticket and mark the issue as a duplicate.
+
+- **Cannot reproduce ecosystem** - Another common issue that can lead to a ticket being closed is that the setup involved in reproducing the problem is complex and specific to your implementation of Cypress. As any developer can imagine, the variety of ecosystems that run Cypress are as varied as the number of flowers in the world. And not surprisingly, we do not have infrastructure setup to mimic every possible scenario. It is possible, even likely, that we are not set up to investigate your specific use case of Cypress. This does not mean that the issue is not real and that it is not important. It simply means we are not equipped to investigate it any further. We are more than happy to point you in the direction of resources that may help you dive into your problem further on your own but we will not be able to replicate your entire stack internally to properly reproduce your problem. Without a consistent reproduction of the issue, it is highly unlikely an issue will be prioritized by a team for inclusion in their sprint work.
+
+- **Stumped, for now** - Sometimes an author provides all the details we could ask for, is very responsive, and uses a straightforward setup - and it still stumps the Cypress devs! The reality is that there are some issues we just can't figure out because of limited resources. We love our large and active community, but in order to support it, sometimes we must move on. We understand this is not something an active issue author wants to hear, but we do not have the resources to put infinite time and energy into every single issue until it is resolved. During the course of our investigation we will attempt to understand the scope of the users being affected by the problem in the issue. If we determine this is an edge case or of low impact to effected users, we will close the issue until such a time that more information comes to light that sheds new insight into the potential root of the problem.
```

Comment:
I think there's a good opportunity here to reword this to encourage the user to submit their own contribution. Maybe something about welcoming contributions

Reply:
i added blurbs about getting folks to contribute